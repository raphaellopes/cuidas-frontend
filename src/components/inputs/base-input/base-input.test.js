// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import BaseInput from './index';

const component = props => (renderer.create(
  <BaseInput {...props} />,
));

describe('<BaseInput />', () => {
  test('Should render the component', () => {
    const tree = component().toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('Should change the color when prop is set', () => {
    const props = { color: 'primary' };
    const tree = component(props).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('Should change the borderColor when prop is set', () => {
    const props = { borderColor: 'primary' };
    const tree = component(props).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
