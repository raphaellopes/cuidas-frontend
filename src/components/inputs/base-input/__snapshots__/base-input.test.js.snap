// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseInput /> Should change the borderColor when prop is set 1`] = `
<input
  className="sc-bdVaJa hyNoUk"
  type="text"
/>
`;

exports[`<BaseInput /> Should change the color when prop is set 1`] = `
<input
  className="sc-bdVaJa bNCxGc"
  color="primary"
  type="text"
/>
`;

exports[`<BaseInput /> Should render the component 1`] = `
<input
  className="sc-bdVaJa dmWOiu"
  type="text"
/>
`;
